custom_components/waste_collection_schedule/waste_collection_schedule/source/reigatebanstead_gov_uk.py

```diff
@@ -40,13 +41,6 @@ def fetch(self):
 
         s = requests.Session()
 
-        # Set up session
-        timestamp = time_ns() // 1_000_000  # epoch time in milliseconds
-        session_request = s.get(
-            f"https://my.reigate-banstead.gov.uk/apibroker/domain/my.reigate-banstead.gov.uk?_={timestamp}",
-            headers=HEADERS,
-        )
-
```

Comment:
same

Reply:
Good point - I did run the full test suite before and after these changes, and didn't see any unintended change.

Reply:
```
python test_sources.py -s reigatebanstead_gov_uk -i -l -t
Testing source reigatebanstead_gov_uk ...
  found 14 entries for Test_001
    2024-05-10 : Food waste [None]
    2024-05-10 : Paper and cardboard [None]
    2024-05-10 : Mixed recycling [None]
    2024-05-10 : Refuse [None]
    2024-05-17 : Food waste [None]
    2024-05-17 : Garden waste [None]
    2024-05-17 : Paper and cardboard [None]
    2024-05-24 : Food waste [None]
    2024-05-24 : Paper and cardboard [None]
    2024-05-24 : Mixed recycling [None]
    2024-05-24 : Refuse [None]
    2024-05-31 : Food waste [None]
    2024-05-31 : Garden waste [None]
    2024-05-31 : Paper and cardboard [None]
  found 14 entries for Test_002
    2024-05-10 : Food waste [None]
    2024-05-10 : Paper and cardboard [None]
    2024-05-10 : Mixed recycling [None]
    2024-05-10 : Refuse [None]
    2024-05-17 : Food waste [None]
    2024-05-17 : Garden waste [None]
    2024-05-17 : Paper and cardboard [None]
    2024-05-24 : Food waste [None]
    2024-05-24 : Paper and cardboard [None]
    2024-05-24 : Mixed recycling [None]
    2024-05-24 : Refuse [None]
    2024-05-31 : Food waste [None]
    2024-05-31 : Garden waste [None]
    2024-05-31 : Paper and cardboard [None]
  found 56 entries for Test_003
    2024-05-09 : Glass [None]
    2024-05-09 : Mixed cans [None]
    2024-05-09 : Plastic [None]
    2024-05-10 : Paper and cardboard [None]
    2024-05-10 : Trade - Refuse [None]
    2024-05-13 : Paper and cardboard [None]
    2024-05-13 : Trade - Paper and cardboard [None]
    2024-05-13 : Trade - Refuse [None]
    2024-05-14 : Glass [None]
    2024-05-14 : Mixed cans [None]
    2024-05-14 : Plastic [None]
    2024-05-15 : Paper and cardboard [None]
    2024-05-15 : Trade - Paper and cardboard [None]
    2024-05-15 : Trade - Refuse [None]
    2024-05-16 : Glass [None]
    2024-05-16 : Mixed cans [None]
    2024-05-16 : Plastic [None]
    2024-05-17 : Paper and cardboard [None]
    2024-05-17 : Trade - Refuse [None]
    2024-05-20 : Paper and cardboard [None]
    2024-05-20 : Trade - Paper and cardboard [None]
    2024-05-20 : Trade - Refuse [None]
    2024-05-21 : Glass [None]
    2024-05-21 : Mixed cans [None]
    2024-05-21 : Plastic [None]
    2024-05-22 : Paper and cardboard [None]
    2024-05-22 : Trade - Paper and cardboard [None]
    2024-05-22 : Trade - Refuse [None]
    2024-05-23 : Glass [None]
    2024-05-23 : Mixed cans [None]
    2024-05-23 : Plastic [None]
    2024-05-24 : Paper and cardboard [None]
    2024-05-24 : Trade - Refuse [None]
    2024-05-27 : Paper and cardboard [None]
    2024-05-27 : Trade - Paper and cardboard [None]
    2024-05-27 : Trade - Refuse [None]
    2024-05-28 : Glass [None]
    2024-05-28 : Mixed cans [None]
    2024-05-28 : Plastic [None]
    2024-05-29 : Paper and cardboard [None]
    2024-05-29 : Trade - Paper and cardboard [None]
    2024-05-29 : Trade - Refuse [None]
    2024-05-30 : Glass [None]
    2024-05-30 : Mixed cans [None]
    2024-05-30 : Plastic [None]
    2024-05-31 : Paper and cardboard [None]
    2024-05-31 : Trade - Refuse [None]
    2024-06-03 : Paper and cardboard [None]
    2024-06-03 : Trade - Paper and cardboard [None]
    2024-06-03 : Trade - Refuse [None]
    2024-06-04 : Glass [None]
    2024-06-04 : Mixed cans [None]
    2024-06-04 : Plastic [None]
    2024-06-05 : Paper and cardboard [None]
    2024-06-05 : Trade - Paper and cardboard [None]
    2024-06-05 : Trade - Refuse [None]
  found 56 entries for Test_004
    2024-05-09 : Glass [None]
    2024-05-09 : Mixed cans [None]
    2024-05-09 : Plastic [None]
    2024-05-10 : Paper and cardboard [None]
    2024-05-10 : Trade - Refuse [None]
    2024-05-13 : Paper and cardboard [None]
    2024-05-13 : Trade - Paper and cardboard [None]
    2024-05-13 : Trade - Refuse [None]
    2024-05-14 : Glass [None]
    2024-05-14 : Mixed cans [None]
    2024-05-14 : Plastic [None]
    2024-05-15 : Paper and cardboard [None]
    2024-05-15 : Trade - Paper and cardboard [None]
    2024-05-15 : Trade - Refuse [None]
    2024-05-16 : Glass [None]
    2024-05-16 : Mixed cans [None]
    2024-05-16 : Plastic [None]
    2024-05-17 : Paper and cardboard [None]
    2024-05-17 : Trade - Refuse [None]
    2024-05-20 : Paper and cardboard [None]
    2024-05-20 : Trade - Paper and cardboard [None]
    2024-05-20 : Trade - Refuse [None]
    2024-05-21 : Glass [None]
    2024-05-21 : Mixed cans [None]
    2024-05-21 : Plastic [None]
    2024-05-22 : Paper and cardboard [None]
    2024-05-22 : Trade - Paper and cardboard [None]
    2024-05-22 : Trade - Refuse [None]
    2024-05-23 : Glass [None]
    2024-05-23 : Mixed cans [None]
    2024-05-23 : Plastic [None]
    2024-05-24 : Paper and cardboard [None]
    2024-05-24 : Trade - Refuse [None]
    2024-05-27 : Paper and cardboard [None]
    2024-05-27 : Trade - Paper and cardboard [None]
    2024-05-27 : Trade - Refuse [None]
    2024-05-28 : Glass [None]
    2024-05-28 : Mixed cans [None]
    2024-05-28 : Plastic [None]
    2024-05-29 : Paper and cardboard [None]
    2024-05-29 : Trade - Paper and cardboard [None]
    2024-05-29 : Trade - Refuse [None]
    2024-05-30 : Glass [None]
    2024-05-30 : Mixed cans [None]
    2024-05-30 : Plastic [None]
    2024-05-31 : Paper and cardboard [None]
    2024-05-31 : Trade - Refuse [None]
    2024-06-03 : Paper and cardboard [None]
    2024-06-03 : Trade - Paper and cardboard [None]
    2024-06-03 : Trade - Refuse [None]
    2024-06-04 : Glass [None]
    2024-06-04 : Mixed cans [None]
    2024-06-04 : Plastic [None]
    2024-06-05 : Paper and cardboard [None]
    2024-06-05 : Trade - Paper and cardboard [None]
    2024-06-05 : Trade - Refuse [None]
```